WooCommerce/Classes/POS/ViewModels/PointOfSaleDashboardViewModel.swift

```diff
@@ -106,12 +110,40 @@ final class PointOfSaleDashboardViewModel: ObservableObject {
     func cardPaymentTapped() {
         Task { @MainActor in
             showsCreatingOrderSheet = true
-            let order = try await createTestOrder()
+            let order = try await createTestOrder(amount: formattedOrderTotalPrice ?? "15.00")
```

Comment:
❓ At this point, when this function is invoked without a connected reader, is the order supposed to be ready? IMO it doesn't make sense for the CTA to be visible until the order is ready for payment. In this case, we probably want to guard on the order like the auto-payment case.

Reply:
Yes, it should be. I just didn't focus on the disconnected-reader flow in this PR.